modules/system/ServiceProvider.php

```diff
@@ -52,6 +52,7 @@ public function register()
          */
         PluginManager::instance()->registerAll();
 
+        $this->registerAliases();
```

Comment:
@mjauvin I'm gonna rename this to `registerClassAliases`, so that it's less likely that it's already in use in plugins.
```suggestion
        $this->registerClassAliases();
```